opentelemetry-contrib/src/lib.rs

```diff
@@ -4,6 +4,16 @@
 //! some users.
 //!
 //! Typically, those include vendor specific propagators.
+//!
+//! ## Crate Feature Flags
+//!
+//! The following crate feature flags are available:
+//!
+//! * `datadog`: Adds a Datadog trace exporter.
+//! * `reqwest-blocking-client`: Export spans using the reqwest blocking http
```

Comment:
Should be `exports` I guess?
```suggestion
//! * `reqwest-blocking-client`: Exports spans using the reqwest blocking http
```

Reply:
Hm this was intended to read more as "If you use this feature it will export spans using x" rather than "the client exports spans using x" as it is a description of the feature rather than what the client does